collection-spec/collection-spec.md

```diff
@@ -19,7 +19,7 @@ STAC Collections are meant to be compatible with *OGC API - Features* Collection
 | --------------- | ------------------------------------------------ | ------------------------------------------------ |
 | stac_version    | string                                           | **REQUIRED.** The STAC version the Collection implements. STAC versions can be mixed, but please keep the [recommended best practices](../best-practices.md#mixing-stac-versions) in mind. |
 | stac_extensions | \[string]                                        | A list of extension identifiers the Collection implements. |
-| id              | string                                           | **REQUIRED.** Identifier for the collection that is unique across the provider. |
+| id              | string                                           | **REQUIRED.** Identifier for the collection that is unique across the provider, ideally globally unique. |
```

Comment:
I'm not sure this works in practice. I don't think providers often want to complicate their collection ids and have something like https://provider.com/collections/provider.com.sentinel-2 - instead they will usually just use sentinel-2, especially when the ids are re-used in user-oriented processing workflows like in openeo.